src/Microsoft.CodeAnalysis.ResxSourceGenerator/Microsoft.CodeAnalysis.ResxSourceGenerator.UnitTests/Resources/SingleString_RootNamespaceVisualBasicAsync/.Resources.Designer.vb

```diff
@@ -0,0 +1,33 @@
+﻿' <auto-generated/>
+
+Imports System.Reflection
+
+
+Namespace Global.
```

Comment:
I'm pretty certain this isn't legal VB.

Reply:
It's not. The test is skipped as not currently supported.